pytest_watch/command.py

```diff
@@ -87,23 +91,24 @@ def main(argv=None):
     if args['--pdb']:
         pytest_args.append('--pdb')
 
-    # Parse extensions
+    # Parse extensions [default: .py]
     if args['--ext'] == '*':
         extensions = ALL_EXTENSIONS
     elif args['--ext']:
         extensions = [('.' if not e.startswith('.') else '') + e
                       for e in args['--ext'].split(',')]
-    else:
-        extensions = None
```

Comment:
Ahh I see fixing the default value in docopt allows you to remove this. Very nice 👍